presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -111,9 +111,9 @@ security options in the Hive connector.
 Hive Configuration Properties
 -----------------------------
 
-================================================== ============================================================ ==========
+================================================== ============================================================ =============================
```

Comment:
We don't need to widen the table now that the default is "false"

Reply:
Right. I'm staill going to widen it for 2 symbols to cover the `RCBINARY`